pkgs/development/tools/nagelfar/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchzip, tcl, tcllib, runtimeShell }:
+
+tcl.mkTclDerivation rec {
+  pname   = "nagelfar";
+  version = "1.3.3";
+
+  src = fetchzip {
+    url    = "https://sourceforge.net/projects/nagelfar/files/Rel_133/nagelfar133.tar.gz";
+    sha256 = "sha256-bdH53LSOKMwq53obVQitl7bpaSpwvMce8oJgg/GKrg0=";
+  };
+
+  buildInputs = [
+    tcl
+    tcllib
+  ];
+
+  dontUnpack    = true;
```

Comment:
This is pretty inconsistent with the code style used in the rest of nixpkgs.

Reply:
updated